windows/vhba_hot_add_remove/nvme_disk_hot_extend_spec13.yml

```diff
@@ -15,7 +15,9 @@
           vars:
             skip_msg: "Skip this test case due to ESXi version '{{ esxi_version }}' is not >= 7.0.2."
             skip_reason: "Not Supported"
-          when: esxi_version is undefined or esxi_version is version('7.0.2', '<')
+          when: >-
+            (esxi_version is undefined or esxi_version is version('7.0.2', '<')) or
+            (vm_hardware_version_num is undefined or vm_hardware_version_num | int < 19)
```

Comment:
It looks like the feature is limited to hardware version >= 19, right? Then here you can just check hardware version, as hardware version >= 19 means ESXi version >= 7.0.2.

Reply:
Yes. We can only limit it to hardware versioin check.